sdk/tests/conformance2/textures/misc/tex-storage-2d.html

```diff
@@ -223,6 +223,9 @@
             // defined texture images with null data, which should sample as RGBA 0,0,0,0.
             gl.texParameteri(target, gl.TEXTURE_MIN_FILTER,
                              testcase.mipmap ? gl.NEAREST_MIPMAP_NEAREST : gl.NEAREST);
+            if (testcase.type == gl.FLOAT) {
```

Comment:
Comment after the fact: I think this would have been better specified as another parameter in the test case, rather than hardwiring just this one format.